packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -108,6 +109,20 @@ abstract class ProgressIndicator extends StatefulWidget {
   /// {@endtemplate}
   final String? semanticsValue;
 
+  /// {@template flutter.material.ProgressIndicator.hasRoundStroke}
+  /// The progress indicator's line ending: square or round.
+  ///
+  /// This determines the shape of the stroke ends of the progress indicator.
+  /// False is the default value, and it results in square ends for the
+  /// progress indicator. The true value results in rounded ends for the
+  /// stroke.
+  ///
+  /// If [ProgressIndicator.hasRoundStroke] is null then it will use the
+  /// ambient [ProgressIndicatorThemeData.hasRoundStroke]. If that is null
```

Comment:
Consider adding a test to check that ProgressIndicatorThemeData fallback work as expected.

Reply:
Thanks. Fixed.